client/keys/utils.go

```diff
@@ -256,3 +257,19 @@ func PostProcessResponse(w http.ResponseWriter, cdc *codec.Codec, response inter
 	w.Header().Set("Content-Type", "application/json")
 	w.Write(output)
 }
+
+// IsKeyNotFoundErr - check if the error means that the specified key doesn't exist
+func IsKeyNotFoundErr(err error, name string) bool {
```

Comment:
Can we export references to the exact error types instead of this sort of logic? It introduces technical debt.

Reply:
I have thought about this idea. But this will change key base implementaion. To avoid too much impact to the code, I chosen this implementation.

Reply:
@HaoyangLiu How so? Ideally the client documentation shows the exact return types.

Reply:
The key base should *export* the error types directly - if it doesn't, let's fix it - all the relevant code should be in this repo I think.

Reply:
I'm sorry for misleading you. All code about key base implementation are in this repo too. My point is to minimize the change and limit the change to client code. If this is not a problem, I will change to code according to your suggestions.
@fedekunze 
The error has no error code or abci code.